internal/batches/executor/executor.go

```diff
@@ -147,20 +148,20 @@ func (x *executor) do(ctx context.Context, task *Task, ui TaskExecutionUI) (err
 	ui.TaskStarted(task)
 
 	// Let's set up our logging.
-	log, err := x.opts.Logger.AddTask(util.SlugForPathInRepo(task.Repository.Name, task.Repository.Rev(), task.Path))
+	l, err := x.opts.Logger.AddTask(util.SlugForPathInRepo(task.Repository.Name, task.Repository.Rev(), task.Path))
```

Comment:
Good catch — I often shadow a package name inadvertently and then wonder why things break later.